gcb/release/cloudbuild.yaml

```diff
@@ -21,7 +21,7 @@ steps:
   dir: "go/src/k8s.io"
   args:
   - "clone"
-  - "--branch=${_TOOL_BRANCH}"
+  - "--branch=${_TOOL_REF}"
```

Comment:
As it stands we will successfully get the value from the substitution here. But I think that this still leaves us only with the possibility to use a specific branch (or a tag). We still need to check out a commit if we want to use an arbitrary ref.